src/polyfills/custom-elements.js

```diff
@@ -265,7 +265,7 @@ class Registry {
      *
```

Comment:
comment is no longer true: "This is cleared out when the mutation observer is created."

Reply:
Done.